examples/test-secrets-in-explicit-provider/csharp/Program.cs

```diff
@@ -1,24 +1,62 @@
-﻿using Pulumi;
+﻿using System;
 using System.Collections.Generic;
+using System.Text;
+
+using Pulumi;
+using Pulumi.Random;
 
 return await Deployment.RunAsync(() =>
 {
-   var provider = new Pulumi.Docker.Provider("docker", new Pulumi.Docker.ProviderArgs
+
+   // Test passing values marked secret by the user program to the provider.
+   var provider1 = new Pulumi.Docker.Provider("provider-with-sensitive-address", new Pulumi.Docker.ProviderArgs
```

Comment:
Which of these providers are currently needing to be skipped in testing? it might be useful to have separate pulumi programs for each, but if the failed test output makes it clear then I see no need.